test/parallel/test-net-connect-immediate-finish.js

```diff
@@ -23,8 +23,8 @@
 const common = require('../common');
 const assert = require('assert');
 const net = require('net');
-
-const client = net.connect({host: '***', port: common.PORT});
+// port 8080 is hardcoded since this does not create a network connection
+const client = net.connect({host: '***', port: 8080});
```

Comment:
It's a good try, but IMHO it's too arbitrary, you could just use `0`.

A (controversial) other way is to copy what we did in [12964](https://github.com/nodejs/node/pull/12964/files#diff-25c120fdbd0b3ef605ab2877a5e523c0R6) but only do that as an exercise!